src/style/app.module.css

```diff
@@ -603,16 +615,48 @@
 
 .addButton {
   margin-bottom: 10px;
-  color: var(--brown-color);
+  color: var(--dark-grey-font);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`